partcad/src/partcad/part_factory_feature_ai.py

```
@@ -116,6 +118,28 @@
         self.instantiate_orig = self.instantiate
         self.instantiate = self._instantiate_ai
 
+    def _add_header(self, script):
+        from partcad import __version__
+        date_string = datetime.now().strftime("%b %d, %Y")
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)